Updated the settings views to use Swift and storyboards

jamie-conlon opened the pull request:
Storyboards
- The initial storyboard is the main settings view. This is a static tableview.
- The second storyboard is the sync settings view. This is a dynamic tableview where we can have various syncing services with different settings associated to each one.

Swift
- Code is pretty much the same as obj-c but just change to Swift syntax.

mgmart commented on Classes/Settings/SettingsController.swift:
```diff
@@ -0,0 +1,113 @@
+//  MobileOrg
+//
+//  Created by Jamie Conlon on 04/01/2017.
+//  Copyright © 2017 Sean Escriva. All rights reserved.
```
Should be © 2017 Jamie Conlon, shouldn't it?

mgmart commented on Classes/Settings/SettingsController.swift:
```diff
@@ -0,0 +1,113 @@
+//  Created by Jamie Conlon on 04/01/2017.
+//  Copyright © 2017 Sean Escriva. All rights reserved.
+//
+
```
GNU License Header missing

mgmart commented on Classes/Settings/SettingsController.swift:
```diff
@@ -0,0 +1,113 @@
+
+class SettingsController: UITableViewController {
+    
+    @IBOutlet weak var appBadgeSwitch: UISwitch!
```
The existing Swift files have a 2 spaces indentation. Would be nice if we could keep this consistent.

webframp commented:
I've used this in the past to define project editor settings: http://editorconfig.org/

Requires a plugin with Xcode though

mgmart commented:
Yeah, and to use plugins you have to unsign Xcode nowadays :(

webframp commented:
You may want this: https://help.github.com/articles/remove-sensitive-data/ 

We can't delete the PR, but if you rewrite what's in the branch this PR is based on you'll be fine

jamie-conlon commented:
I deleted the whole branch and created a new branch with the same name (I know this was pretty stupid but I was very confused at this stage!) So I don't think I'll be able to rewrite over this PR... 
I've changed my dropbox api keys so thats not really an issue. 

My new PR should have no sensitive info in it. I think...
git 1 - Jamie 0...
